src/components/RegisterForm/RegisterForm.tsx

```diff
@@ -78,6 +82,15 @@ export const RegisterForm: FC = () => {
       return axios.post<IGeneralPostResponse>(`/api/user/registration?seminar=${seminar}`, transformFormData(data))
     },
     onSuccess: () => router.push(`${router.asPath}/../verifikacia`),
+    onError: (error: AxiosError<RegisterErrorResponseData>) => {
+      if (error.response?.status === 400) {
+        if (error.response.data.email) {
+          setError('email', {type: 'custom', message: `* ${error.response.data.email[0]}`})
```

Comment:
``* A USER IS ALREADY REGISTERED WITH THIS E-MAIL ADDRESS.`

ale uz som sa dohodol s BE ze nam poslu slovensky message, takze je to v poriadku